sdk/iOS/src/MSOperationQueue.m

```diff
@@ -60,7 +60,7 @@ -(NSArray *) getOperationsForTable:(NSString *)table item:(NSString *)item
     MSQuery *query = [[MSQuery alloc] initWithSyncTable:syncTable];
     
     if (item) {
-        query.predicate = [NSPredicate predicateWithFormat:@"(table == %@) AND (itemId == %@)", table, item];
+        query.predicate = [NSPredicate predicateWithFormat:@"(table == %@) AND (itemId ==[c] %@)", table, item];
```

Comment:
There are cons of treating table name in a case insensitive manner. If we think that is needed let us take it as a separate work item.